src/java/test/org/apache/zookeeper/common/NetUtilsTest.java

```diff
@@ -0,0 +1,46 @@
+package apache.zookeeper.common;
```

Comment:
???

Reply:
@mjeelanimsft i think this is why the tests are failing. can you fix this package name? (good catch @maoling !)

Reply:
Thanks @maoling @breed - working on fixing this